scylla-cql/src/frame/value_tests.rs

```diff
@@ -650,12 +650,46 @@ fn tuple_batch_values() {
 fn ref_batch_values() {
     let batch_values: &[&[i8]] = &[&[1, 2], &[2, 3, 4, 5], &[6]];
 
-    assert_eq!(<&&&&[&[i8]] as BatchValues>::len(&&&&batch_values), 3);
+    return check_ref_bv::<&&&&&[&[i8]]>(&&&&batch_values);
```

Comment:
One ampersand too many in `&&&&&[&[i8]]`?

Reply:
batch_values already has one reference level so I think this is correct